src/poly/orderings.jl

```diff
@@ -102,9 +110,17 @@ ordering_lp(nvars::Int = 1) = _basic_ordering(Singular.ringorder_lp, nvars)
     ordering_rp(nvars::Int = 1)
 
 Represents a block of at least `nvars` variables with the
-reverse lexicographical ordering (:revlex).
+inverse lexicographical ordering (:invlex).
 """
-ordering_rp(nvars::Int = 1) = _basic_ordering(Singular.ringorder_rp, nvars)
+ordering_rp(nvars::Int = 1) = _basic_ordering(Singular.ringorder_ip, nvars)
+
+@doc raw"""
+    ordering_ip(nvars::Int = 1)
+
+Represents a block of at least `nvars` variables with the
+inverse lexicographical ordering (:invlex).
```

Comment:
I don't mind having these docstrings added, but let me point out that they are irrelevant for OSCAR; what we really want are such docstrings *there* (if we don't have them already).

Regarding the renaming of "rp" to "ip" I already objected elsewhere; I would suggest to keep it here, too, and not add a second variant. In the text, I don't think it is helpful to say either "inverse lexicographical ordering" or "reverse lexicographical ordering" alone, as that still doesn't clarify which definition is used.

It is also unclear to a casual what the `(:invlex)` here refers to.

How about something like this, borrowing from the Singular manual a bit:
```suggestion
Represents a block of at least `nvars` variables with the Singular ordering `rp`.
This stands for what the Singular manual refers to as "reverse lexicographical ordering" (and is elsewhere sometimes called "inverse lexicographical ordering), i.e. a lexicographical ordering from the right with `1 < x_1 < ... <x_n`.

Note that this reverses the "natural" order `x_1 > ... > x_n`.

When using `polynomial_ring`, this ordering can be selected via `ordering=:invlex`.
```
(Note that I left ":invlex" in the final line, but maybe it should be `:revlex`, or maybe it should be `:rp`...)